docs/README.md

```diff
@@ -1,14 +1,20 @@
-# VTEX Country Flags
+📢 Use this project, [contribute](https://github.com/{OrganizationName}/{AppName}) to it or open issues to help evolve it using [Store Discussion](https://github.com/vtex-apps/store-discussion).
 
-A React component that renders country flags as SVG icons.
+# VTEX COUNTRY FLAGS
 
-## API
+⚠️ **This is an ongoing, unsupported, unfinished and undocumented project. We do not guarantee any results after installation.** ⚠️
 
-### CountryFlag
+<!-- DOCS-IGNORE:start -->
+<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+[![All Contributors](https://img.shields.io/badge/all_contributors-0-orange.svg?style=flat-square)](#contributors-)
+<!-- ALL-CONTRIBUTORS-BADGE:END -->
+<!-- DOCS-IGNORE:end -->
 
-This component receives the country code and renders that respective country's flag.
 
-#### Props
+This component receives the country code and renders that respective country's flag in SVG.
+
+## Configuration 
+
 
```

Comment:
In this section, you first must **add the primary instructions** that will allow users to use the app's blocks in their store, such as:

1. Adding the app as a theme dependency in the `manifest.json` file;
2. Declaring the app's main block in a given theme template or inside another block from the theme.

Remember to add a table with all blocks exported by the app and their descriptions. You can verify an example of it on the [Search Result documentation](https://vtex.io/docs/components/all/vtex.search-result@3.56.1/). 

Next, add the **props table** containing your block's props. 

If the app exports more than one block, create several tables - one for each block. For example:

### `block-1` props

| Prop name | Type | Description | Default value |
| :---------: | :----: | :----------: | :------------: | 
| `XXXXX`  | `XXXXXX` | `XXXXXXX` | `XXXXXX` |